tools/runqslower.py

```diff
@@ -80,7 +83,7 @@
 // record enqueue timestamp
 static int trace_enqueue(u32 tgid, u32 pid)
```

Comment:
could you change all tgid => pid and pid => tid in both python and C code? This will make code cleaner, less confusion and easy to follow.

Reply:
I was also confused when I was writing the code and switched back and forth the naming of the pids/tids/tgids etc. (Because of the kernel  function interfaces) Finally the `tgid` made is the most obvious as the 'g' signalled that it is a group leader, meaning a pid :) But I can understand the argument for keeping pid and tid.